charts/seed-bootstrap/templates/grafana/grafana-datasources-configmap.yaml

```diff
@@ -26,3 +26,9 @@ data:
       isDefault: true
       version: 1
       editable: false
+    - name: loki
+      type: loki
+      access: proxy
+      url: http://loki.garden.svc:3100
+      jsonData:
+        maxLines: 1000
```

Comment:
We can increase it to 5000, because I think 1000 log lines sometimes are not enough to trace the logs

Reply:
Yes, 1000 is a bit small, on the other hand we have it on 10k in the testmachinery and when you really view 10k logs in Grafana, then your browser tab is nearly unresponsive (probably due to the big table rendered in Grafana), so 5k could be a compromise but it shouldn't be bigger than that.